packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuAnchor.test.tsx

```diff
@@ -22,6 +24,7 @@ describe('ContextualMenuButton', () => {
           index={ 0 }
           focusableElementIndex={ 0 }
           totalItemCount={ 1 }
+          onItemClick={ itemOnClick }
```

Comment:
Because the snapshot has it has a function, The actual behavior should be if we have onclick then we should invoke otherwise we shouldnt have any default method.